sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2FunctionSuite.scala

```diff
@@ -428,6 +430,22 @@ class DataSourceV2FunctionSuite extends DatasourceV2SQLBase {
     }
   }
 
+  test("SPARK-37957: pass deterministic flag") {
```

Comment:
Can we make the test name more clear? `pass deterministic flat to what`

Reply:
Sure